candle-datasets/Cargo.toml

```diff
@@ -9,6 +9,9 @@ categories.workspace = true
 license.workspace = true
 readme = "README.md"
 
+[lib]
+crate-type=["dylib"]
```

Comment:
Why do we want this rather than the default?

Reply:
I need to remove, but that's required to be able to detect bloat :)